ESP232.ino

```diff
@@ -1,16 +1,27 @@
+#if defined(ESP8266)
+#pragma message "ESP8266 stuff happening!"
```

Comment:
please remove these lines, they are a bit distacting and make one think theres a compile error

Reply:
looks resolved now